README.md

```diff
@@ -260,4 +260,10 @@ To edit files using Sublime in on the Beaglebone, there is a package in Sublime
 
 http://blog.keyrus.co.uk/editing_files_on_a_remote_server_using_sublime.html
 
+## Errata
+In hardware version 3 MITM, there are a few issues to address:
```

Comment:
as a reader, I'm not sure which hardware revision this is referring. You should clarify, the README.md discusses the truck cape. This passage refers to MITM revs.

For consistency, perhaps you should call them "TruckCape v3 MITM" and "TruckCape v4 MITM" to match the filenames of pdfs you put in docs.